src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -35,7 +35,7 @@ def compress(self, model):
         for name, module in model.named_modules():
             layer = LayerInfo(name, module)
             config = self._select_config(layer)
-            if config is not None:
+            if config is not None and layer.type in config['op_types']:
```

Comment:
_select_config already checked whether `layer.type in config['op_types']`?

Reply:
Good catch, removed.